lib/uploadcare/rails/active_record/mount_uploadcare_file.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'active_support/concern'
+
+module Uploadcare
+  module Rails
+    # A module containing ActiveRecord extension. Allows to use uploadcare file methods in Rails models
+    module ActiveRecord
+      extend ActiveSupport::Concern
+
+      UUID_REGEX = /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/.freeze
+
+      def build_uploadcare_file(attribute)
+        cdn_url = attributes[attribute.to_s].to_s
+        return nil if cdn_url.empty?
+
+        uuid = cdn_url.match(UUID_REGEX).to_s
+        ::Rails.cache.read(cdn_url) || Uploadcare::Rails::File.new(cdn_url: cdn_url, uuid: uuid.presence)
```

Comment:
I think, we need some namespace for the cache key (`['uploadcare', cdn_url]`). It can be configurable.

Reply:
Also, `cache.read` only reads values from the cache, doesn't write. Is this OK?

Reply:
Can we safely save `Uploadcare::Rails::File` objects to the cache (serialization issues etc)?

Reply:
> Also, `cache.read` only reads values from the cache, doesn't write. Is this OK? Should we use `cache.fetch` instead?

Yes, it is ok, but I find `fetch` more suitable in this case :+1:

Reply:
> Can we safely save `Uploadcare::Rails::File` objects to the cache (serialization issues etc)?

No issues occurred during testing.

Reply:
It uses `Marshal` inside, it would be better to store just hash.

Reply:
As I understand, if we store just an attribute hash - we have to create a new Uploadcare::Rails::File file each time. But the purpose of caching here is to avoid file creating each time. Right?

Reply:
If the cache here just helps to avoid creating of a small in-memory object, then the cache is not needed.
It is needed to get rid of lengthy computations, network requests, database queries, etc.